code/script.js

```diff
@@ -253,91 +257,144 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  winOrLoseSection.classList.remove('show-win-or-lose-wrapper')
+  generateBoard()
+  setSecret()
+  selectQuestion()
+  questions.value = 'defaultOption' 
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
+  const optionValue = questions.value;
 
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
+      value: optionValue,
       category: category,
     }
   } else if (category === 'eye color') {
-    // Set this up your self
+    currentQuestion = {
+    attribute: 'eyeColor',
+    value: optionValue,
+    category: category,
```

Comment:
remember that you do not need to add value when you want to pass the variable with the same name as the key. 
`{category: category}` can be written as `{category}` 👍🏻 And when you have a bigger project you can use saved time to watch some Netflix 🙈